test/language/statements/class/elements/privatefield-on-proxy.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Successfully access private field on Proxy objects
```

Comment:
I believe these descriptions should extend to something like: _Sucessyfully get private reference without using [[Get]]`_

and the part using `[[Get]]` could be used in the info:

```
Return ? base.[[Get]](GetReferencedName(V), GetThisValue(V)).
```